client/components/signup-form/index.jsx

```diff
@@ -65,6 +67,7 @@ class SignupForm extends Component {
 		goToNextStep: PropTypes.func,
 		handleSocialResponse: PropTypes.func,
 		isSocialSignupEnabled: PropTypes.bool,
+		isSocialFirst: PropTypes.bool,
```

Comment:
Super minor, but this variable could be more explicit (e.g. `isSocialSignupFirst`). Let's also try to keep this list of prop types ordered alphabetically.

Reply:
👍